packages/flutter/test/animation/animations_test.dart

```diff
@@ -238,8 +238,23 @@ void main() {
       vsync: const TestVSync(),
     );
     final CurvedAnimation curved = CurvedAnimation(parent: controller, curve: BogusCurve());
-
-    expect(() { curved.value; }, throwsFlutterError);
+    FlutterError error;
+    try {
+      curved.value;
+    } on FlutterError catch (e) {
+      error = e;
+    }
+    expect(error, isNotNull);
+    expect(error.toStringDeep(), matches(
```

Comment:
add a comment that a RegExp is required due to flutter web and flutter mobile generating slightly different floating point numbers.